vivisect/tests/testvivisect.py

```diff
@@ -46,6 +46,19 @@ def test_xrefs_types(self):
                 self.assertEqual((xflags, isint(xflags)),
                                  (xflags, True))
 
+    def test_cli_searchopcode(self):
+        '''
+        Test that VivCli.do_searchopcodes works
+        '''
+        self.chgrp_vw.do_searchopcodes('ret')
+        self.chgrp_vw.do_searchopcodes('foo')
+        self.chgrp_vw.do_searchopcodes('-f 0x08050200 ret')
+        self.chgrp_vw.do_searchopcodes('-c ret')
+        self.chgrp_vw.do_searchopcodes('-o ret')
+        self.chgrp_vw.do_searchopcodes('-t ret')
+        self.chgrp_vw.do_searchopcodes('-M red ret')
+        self.chgrp_vw.do_searchopcodes('-R r.t')
```

Comment:
there's nothing to assert.  they just write directly to the gui console window.  
the test is that nothing blows up.  agreed that it's not ideal, but we had no tests and we missed that this was broken.  this is me trying to add a test that wouldn't allow that to happen again.

Reply:
So I would at least say mark this as a Todo, because I dislike a test just being "do we crash".

I get that we don't have a good way to test the UI, but I feel like there is a good way that we can figure out, and I don't want to lose this when we inevitably get to that stage.

Reply:
sure.